### PR TITLE
chore: add PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by marcoscaceres**
@yoavweiss and other folks, WDTY? Anything else we should check or add to this template?

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+For normative spec changes, please fill out the following - otherwise, please delete the template below.
+
+Closes #???
+
+Implementation commitment:
+
+- [ ] [WebKit](https://bugs.webkit.org/???)
+- [ ] [Chromium] (https://bugs.chromium.org/???)
```
Comment (marcoscaceres) [inline .github/PULL_REQUEST_TEMPLATE.md:8]:
```suggestion
- [ ] [Chromium](https://bugs.chromium.org/???)
```

**Comment by yoavweiss** [comment]
Let me discuss this with the group on this week's call and we'll see what folks say.

**Comment by marcoscaceres** [comment]
That be great! Thank you!

**Comment by nicjansma** [comment]
Suggestions from [W3C call](https://docs.google.com/document/d/1FMPaDajerE6Qp1dyo6aAgUfwnatOTEMZ3DqSeYzHHW0/edit#):

* Changing `BREAKING CHANGE:` to just `change:`
* Adding a new question about whether this would have substantial breaking changes and to explain them
* Move the PR types to the top (chore/editorial/change), describing what each is
* Noting some of the subsequent questions may not apply to all types, e.g. `Implementation commitment` only for `change:`s, especially breaking ones
* Noting that not all fields of the template need to be 100% complete when opening, but required ones should be filled in before merge
* `For normative spec changes, please fill out the following - otherwise, please delete the template below.` may be confusing for `chore` and `editorial`, as we still want them to fill in some details for those

**Comment by marcoscaceres** [comment]
Thanks for the great feedback @nicjansma! I've tried to integrate all your suggestions. Looking pretty good. Please feel free to jump in and make any additional edits or change suggestion directly on GitHub.

**Review by yoavweiss — APPROVED** [review]
Touched up the text a bit. LGTM!!

**Comment by marcoscaceres** [comment]
@yoavweiss, good to merge?

**Comment by yoavweiss** [comment]
Yup!!

**Pull request merged**